### ISSUE TITLE
Core Voltage SVI2 same Sensor as HWInfo?

### ISSUE THREAD

**Issue by CaptainCoward** [ocerman/zenmonitor#39]
I know that might be a silly question. But i currently investigating some Issue i have on Linux and i suspect it has to do with different Voltages Linux uses. 
I discovered that the Voltages on the  "Core Voltage SVI2 TFN" Sensor on HWInfo seems to report a higher Votage than the Core Voltage SVI2 Sensor on Zenmonitor (Ryzen 5000). I now wonder do these tools use the same sensor?

**Comment by KeithMyers** [comment]
Yes, they should.  The SVI2 sensors is the direct telemetry coming of the VRM's supplying the voltages directly to the cpu core and SoC core.
More likely the differences is due the boosting algorithm differences between Windows and Linux.  The Windows OS is more highly tuned to extract the very last bit of performance out of the cpu.
The Linux OS is more likely to leave the cpu in mildly optimized or Balanced boosting algorithm.  The core voltages would likely be lower.
